pkg/signup/signup.go

```diff
@@ -40,6 +40,12 @@ type Signup struct {
 	// Company from the Identity Provider
 	Company string `json:"company"`
 	Status  Status `json:"status,omitempty"`
+	// StartDate is the date that the user's current subscription started, in ISO8601 format
+	StartDate string `json:"startDate,omitempty"`
+	// End Date is the date that the user's current subscription will end, in ISO8601 format
```

Comment:
You are still mentioning the `ISO8601`, but it's actually `RFC3339`

Reply:
Fixed